src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -26,12 +28,18 @@
 class ContentRouteProvider implements RouteProviderInterface
 {
     /**
+     * @var ContentMapperInterface
+     */
+    private $contentMapper;
+
+    /**
      * @var RequestAnalyzerInterface
      */
     private $requestAnalyzer;
 
-    public function __construct(RequestAnalyzerInterface $requestAnalyzer)
+    public function __construct(ContentMapperInterface $contentMapper, RequestAnalyzerInterface $requestAnalyzer)
```

Comment:
Why do we introduce the `ContentMapper` here? How was the content loaded before? It's not really a good idea to inject deprecated stuff, since we want to get rid of it :confused: